usaspending_api/database_scripts/matview_generator/universal_transaction_matview.json

```diff
@@ -144,6 +145,13 @@
     }, {
       "name": "total_obl_bin",
       "columns": [{"name": "\"total_obl_bin\""}]
+    }, {
+      "name": "total_subsidy_cost",
+      "where": "\"total_subsidy_cost\" IS NOT NULL",
```

Comment:
Will this ever be null?

Reply:
Yes it can be when dealing with non-loan type transactions.